src/app/public/src/modules/shared/route-metadata.service.ts

```diff
@@ -19,11 +19,19 @@ export class StacheRouteMetadataService {
     @Inject(STACHE_ROUTE_METADATA_SERVICE_CONFIG)
     public metadata: any[]) {
       this.metadata.forEach(route => {
-        if (route.order) {
-          route.order = numberConverter(route.order);
+        if (route.hasOwnProperty('order')) {
+          route = this.validateNavOrder(route);
         }
       });
     }
+
+  private validateNavOrder(route: any): any {
+    route.order = numberConverter(route.order);
+    if (route.order <= 0 || route.order !== parseInt(route.order, 10)) {
```

Comment:
What is `route.order !== parseInt(route.order, 10)` checking that `numberConverter` didn't already check?